src/SFA.DAS.EmployerIncentives.Commands/Services/IncentivePaymentProfilesService.cs

```diff
@@ -1,39 +1,35 @@
     {
-        private readonly ApplicationSettings _applicationSettings;
-
-        public IncentivePaymentProfilesService(IOptions<ApplicationSettings> applicationSettings)
-        {
-            _applicationSettings = applicationSettings.Value;
-        }
-
-        public Task<IEnumerable<Domain.ValueObjects.IncentivePaymentProfile>> Get()
-        {
-            if (_applicationSettings?.IncentivePaymentProfiles == null)
-            {
-                return Task.FromResult(Enumerable.Empty<Domain.ValueObjects.IncentivePaymentProfile>());
-            }
-            return Task.FromResult(_applicationSettings.IncentivePaymentProfiles.Select(x =>
-                new Domain.ValueObjects.IncentivePaymentProfile(new Domain.ValueObjects.IncentivePhase(x.IncentivePhase),
-                    MapToDomainPaymentProfiles(x.PaymentProfiles).ToList())));
-        }
-
-        private IEnumerable<Domain.ValueObjects.PaymentProfile> MapToDomainPaymentProfiles(List<Infrastructure.Configuration.PaymentProfile> paymentProfiles)
+        public IEnumerable<IncentivePaymentProfile> Get()
         {
-            if (paymentProfiles == null)
+            return new List<IncentivePaymentProfile>
             {
-                return Enumerable.Empty<Domain.ValueObjects.PaymentProfile>();
-            }
-            return paymentProfiles.Select(x => new Domain.ValueObjects.PaymentProfile(x.IncentiveType, x.DaysAfterApprenticeshipStart, x.AmountPayable));
+                new IncentivePaymentProfile(new IncentivePhase(Phase.Phase1),
+                    new List<PaymentProfile>
+                    {
+                        new PaymentProfile(IncentiveType.UnderTwentyFiveIncentive, daysAfterApprenticeshipStart: 89, amountPayable: 1000),
+                        new PaymentProfile(IncentiveType.UnderTwentyFiveIncentive, daysAfterApprenticeshipStart: 364, amountPayable: 1000),
+                        new PaymentProfile(IncentiveType.TwentyFiveOrOverIncentive, daysAfterApprenticeshipStart: 89, amountPayable: 750),
+                        new PaymentProfile(IncentiveType.TwentyFiveOrOverIncentive, daysAfterApprenticeshipStart: 364, amountPayable: 750)
+                    }),
+                new IncentivePaymentProfile(new IncentivePhase(Phase.Phase2),
+                    new List<PaymentProfile>
+                    {
+                        new PaymentProfile(IncentiveType.UnderTwentyFiveIncentive, daysAfterApprenticeshipStart: 89, amountPayable: 1500),
+                        new PaymentProfile(IncentiveType.UnderTwentyFiveIncentive, daysAfterApprenticeshipStart: 364, amountPayable: 1500),
+                        new PaymentProfile(IncentiveType.TwentyFiveOrOverIncentive, daysAfterApprenticeshipStart: 89, amountPayable: 1500),
+                        new PaymentProfile(IncentiveType.TwentyFiveOrOverIncentive, daysAfterApprenticeshipStart: 364, amountPayable: 1500)
+                    }),
+            };
```

Comment:
These need to be in the phase 1 and phase 2 incentive classes, respectively.  That way, everything about the scheme/phase is contained in a single place. IncentivePaymentProfilesService would then be redundant.